packages/nextjs-server/src/indexers.ts

```diff
@@ -22,7 +23,7 @@ export const appIndexer = (
       const inputAppDir = resolve(__dirname, '../template/app');
       const inputGroupDir = join(inputAppDir, 'groupLayouts');
       const inputStorybookDir = join(inputAppDir, 'storybook-preview');
-      const appDir = join(process.cwd(), 'app');
+      let appDir = getAppDir() ?? join(getSrcDir(), 'app');
```

Comment:
Seems unnecessary. Won't `getAppDir` handle src?

Reply:
`getAppDir` returns `undefined` when there is no app dir. This is handling an edge case where the app dir has been deleted between the time the process starts and the indexer runs. I also thought about throwing an error in this case.